book.bib

```diff
@@ -1457,4 +1457,24 @@ @article{Schloss2023
   year= {2023},
   publisher = {Cold Spring Harbor Laboratory},
   doi = {10.1101/2023.06.23.546313}
-}
\ No newline at end of file
+}
+
+@article {Abdill2023,
+	author = {Richard J. Abdill and Samantha P. Graham and Vincent Rubinetti and Frank W. Albert and Casey S. Greene and Sean Davis and Ran Blekhman},
+	title = {Integration of 168,000 samples reveals global patterns of the human gut microbiome},
+	elocation-id = {2023.10.11.560955},
+	year = {2023},
+	doi = {10.1101/2023.10.11.560955},
+	publisher = {Cold Spring Harbor Laboratory},
+	URL = {https://www.biorxiv.org/content/early/2023/10/11/2023.10.11.560955},
+	eprint = {https://www.biorxiv.org/content/early/2023/10/11/2023.10.11.560955.full.pdf},
+	journal = {bioRxiv}
+}
+
+@Manual{compendiumpackage,
+  title = {MicroBioMap: Access the microbiome compendium from R},
```

Comment:
Put MicroBioMap in curly brackets, also "R". Then the rendering keeps the relevant uppercases.